### ISSUE TITLE
FNX-5555 ⁃ Large font size compatibility

### ISSUE THREAD

**Issue by vesta0** [mozilla-mobile/fenix#2265]
Please set the device font to large and test Fenix to identify any bugs as result.

**Comment by bifleming** [comment]
@vesta0 I'm confused about where this belongs...should it not be in the sprint?

**Comment by vesta0** [comment]
@bifleming this is more of a meta, individual listed under this epic are/should be in the sprint.
